torque-core/src/main/kotlin/com/workday/torque/TestRunFactory.kt

```diff
@@ -61,34 +59,18 @@ class TestRunFactory {
                                 "${testSession.failedCount} failed, took " +
                                 "${testSession.durationMillis.millisToHoursMinutesSeconds()}."
                     )
+                    pullDeviceFiles(args, filePuller)
```

Comment:
If we only pull after all is done, wouldn't we pull only the last video and miss any other?

Reply:
the videos are stored in their respective test method specific folders, so you'd have all of them in the end